src/components/pages/order/Main/Admin/AdminPanels.js

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 import OrderContext from "../../../../../context/OrderContext";
 import { theme } from "../../../../../theme";
 import { EMPTY_PRODUCT } from "../../../../enums/product";
-import { getTabIndexSelected, getTabsConfig } from "./tabsConfig";
+import { getTabIndexSelected, getTabsConfig } from "./TabsConfig";
```

Comment:
```suggestion
import { getTabIndexSelected, getTabsConfig } from "./tabsConfig";
```

Ce n'est pas un composant mais un fichier de config -> "t" minuscule.

Reply:
Ah oui bien-sûr tu as raison... Une faute d'inattention de ma part. Merci de me le rappeler.